openfga/v1/openfga_service.proto

```diff
@@ -1180,6 +1180,7 @@ message ReadRequest {
   string continuation_token = 4 [
     json_name = "continuation_token",
     (validate.rules).string.max_bytes = 5120,
+    (validate.rules).string.pattern = "^[A-Za-z0-9-_]+={0,2}$",
```

Comment:
This pattern matches the description [in the RFC](https://datatracker.ietf.org/doc/html/rfc4648#section-4) (and the [stated encoding from go's base64 package](https://cs.opensource.google/go/go/+/refs/tags/go1.23.4:src/encoding/base64/base64.go;l=123))